ipykernel/debugger.py

```diff
@@ -562,6 +564,24 @@ async def richInspectVariables(self, message):
         reply["success"] = True
         return reply
 
+    async def modules(self, message):        
+        modules = [sys.modules[name] for name in sys.modules]
```

Comment:
Any reasons not to use `sys.modules.values()` here ?

Reply:
sys.modules.values() works. thx.